Sources/WalletConnectPush/Client/Wallet/WalletPushClient.swift

```diff
@@ -45,6 +46,7 @@ public class WalletPushClient {
          proposeResponder: PushRequestResponder,
          pushMessageSubscriber: PushMessageSubscriber,
          subscriptionsProvider: SubscriptionsProvider,
+         pushMessagesProvider: PushMessagesProvider,
          deletePushSubscriptionService: DeletePushSubscriptionService,
          deletePushSubscriptionSubscriber: DeletePushSubscriptionSubscriber,
          resubscribeService: PushResubscribeService) {
```

Comment:
resubscribeService unused in PushClient

Reply:
it is in use, it's calling setUpResubscription func after initialisation